cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,50 @@
+describe("User Authentication", () => {
+  describe("Login and Profile Access", () => {
+    it("allows the user to log in and access their profile", () => {
```

Comment:
This is such a BS test. The test user doesn't not fill in the form at all but can submit the form. This is not a real test.

Reply:
The users dont fill inn because placeholder holds valid credentials. We could have added a test with a blank form where the test could try other valid credentials but it seemed redudant.